trlx/trainer/accelerate_ppo_trainer.py

```diff
@@ -325,14 +325,12 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
             else:
                 scores = all_scores[0].clone().detach()
 
-            str_samples, str_prompts, str_outputs = self.decode(prompt_tensors, samples, append_eos_token=True)
```

Comment:
I don't find (from the current main, and from previously merged PRs) any superflous `<eos>` tokens added through the `decode` call here. But probably skipping it is excessive, since it still cuts `stop_sequences`